packages/components/addon/components/hds/dropdown/index.hbs

```diff
@@ -8,14 +8,17 @@
     }}
   </:toggle>
   <:content as |c|>
-    <ul class={{this.listClassNames}} {{style width=@width}}>
+    <ul class={{this.listClassNames}} {{style width=@width}} role="listbox">
```

Comment:
This role would need to be applied only when using the `Checkmark` contextual component. This raises the usual challenge of a parent component knowing about children.

Reply:
Have you tried using a conditional here? 

```
role={{if hasCheckbox "listbox" null }}
```

Ember won't render an attribute at all if the value is `null`.

Reply:
yes, using a boolean here would be an option, just have to make sure we either document it well or we try to determine it automatically within the component